pkg/controller/template/render.go

```diff
@@ -36,7 +35,6 @@ type RenderConfig struct {
 	*mcfgv1.ControllerConfigSpec
 	PullSecret                 string
 	InternalRegistryPullSecret string
-	FeatureGateAccess          featuregates.FeatureGateAccess
```

Comment:
I don't think the template/render controller controls any FG'ed items and there shouldn't be any plans to immediately, so I'm fine with this (but please correct me if I'm missing something)